client/components/UserDecalPanel.js

```diff
@@ -87,25 +109,31 @@ class UserDetailsPanel extends Component {
     return null
   }
 
+  renderDecalCode = (decal) => (
+    <div className="decal" key={decal.id}>
+      <div className="code">
+        <button
+          aria-label="Show decal code"
+          className="icon toggle"
+          name={decal.id}
+          onClick={this._handleToggle}
+          type="button">
+          <FontAwesomeIcon icon={this.state.decalsVisible[decal.id] ? 'eye' : 'eye-slash'} fixedWidth />
+        </button>
+        {this.state.decalsVisible[decal.id] ? decal.attributes.code : `•••••-•••••-•••••-•••••-${decal.attributes.code.substring(24)}`}
```

Comment:
Could we multiline long ternary expressions like this?

```suggestion
        {
          this.state.decalsVisible[decal.id] 
            ? decal.attributes.code 
            : `•••••-•••••-•••••-•••••-${decal.attributes.code.substring(24)}`
        }
```